tutorials/kubernetes.md

```diff
@@ -11,37 +13,6 @@ so we can keep kubelet running inside container (as well as directly on the host
 
 Below, you can find an instruction how to switch one or more nodes on running kubernetes cluster from docker to CRI-O.
 
-### Preparing crio
```

Comment:
Is this section no longer needed?  I'm not seeing it in a new page somewhere.

Reply:
It was largely redundant with setup.md, except it added some more details on configuring runc. I'm more inclined to move the missing information to setup.md, so that redundancy isn't there.

Reply:
Did the runc stuff from below get moved somewhere?  I'm not seeing it?   I'll let you @mrunalp make the call, but I think this step of checking for the required prelim files should be somewhere.